packages/core/test/oracle/Staker.js

```diff
@@ -106,6 +107,53 @@ describe("Staker", function () {
       const stakingBalance = await staker.methods.voterStakes(account1).call();
       assert.equal(stakingBalance.activeStake, amountToStake.add(toWei("640")));
     });
+
+    it("Stake to", async function () {
+      await staker.methods.stakeTo(account2, amountToStake).send({ from: account1 });
```

Comment:
consider adding a check before this that `account2` has nothing before and after they have `amountToStake`. check that `account1` has nothing before and nothing after